### PR TITLE
Add delete and add features to UG

### PR DESCRIPTION
**Pull request by sueann-chua**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -11,6 +11,30 @@ transferable to other devices too!
...
+<u>**Add a candidate: `add c`**</u>
+
+Adds a candidate to the list of candidates.
+
+<u>Format:</u>
+
+    add c name=<NAME> email=<EMAIL> phone=<PHONE_NUMBER>
+
+<u>Example:</u><br/><br/>
+`add c name=<Bryan Seah> email=<bsah@gmail.com> phone=<12345678>` adds a candidate named Bryan Seah
```
Comment (JeffZincatz) [inline docs/UserGuide.md:24]: The linebreak here seems a little out of place. Perhaps linebreak before "adds a ..."?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -11,6 +11,30 @@ transferable to other devices too!
...
+<u>**Delete a candidate: `delete c`**</u>
+
+Deletes a candidate to the list of candidates.
+
+<u>Format:</u>
+
+    delete c <INDEX>
+
+<u>Example:</u><br/><br/>
+`delete c 3` deletes the 3rd candidate from the list of candidates.
```
Comment (JeffZincatz) [inline docs/UserGuide.md:36]: The linebreak here also. Perhaps linebreak before "deletes the ..."?

**Review by JeffZincatz — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (sueann-chua) [reply docs/UserGuide.md]: Noted, made edits accordingly.

File: docs/UserGuide.md (hunk 2 above)
Comment (sueann-chua) [reply docs/UserGuide.md]: Noted, made edits accordingly.

**Review by JeffZincatz — APPROVED** [review]
LGTM.

**Pull request merged**